code/nat-configs/node0-config.toml

```diff
@@ -0,0 +1,64 @@
+moniker = "validator-0"
+
+[logging]
+log_level = "debug"
+log_format = "plaintext"
+
+[consensus]
+timeout_propose = "3s"
+timeout_propose_delta = "500ms"
+timeout_prevote = "1s"
+timeout_prevote_delta = "500ms"
+timeout_precommit = "1s"
+timeout_precommit_delta = "500ms"
+timeout_rebroadcast = "5s"
+value_payload = "proposal-and-parts"
+queue_capacity = 100
```

Comment:
```suggestion
queue_capacity = 100
enabled = true
```

Reply:
Needed for all, after feat(CCHAIN-274): Allow nodes to run in sync-only mode without participating in consensus (#1174)

Reply:
try to update, we fixed the config default in main.